test/runtime/connectivity.go

```diff
@@ -93,6 +93,15 @@ var runtimeConnectivityTest = func(datapathMode string) func() {
 					return true
 				}, "Endpoints are not ready", &helpers.TimeoutConfig{Timeout: 150 * time.Second})
 				Expect(err).Should(BeNil())
+
+				err = helpers.WithTimeout(func() bool {
+					res := vm.ContainerExec(helpers.Server, "netperf -H 127.0.0.1 -l 1")
+					if !res.WasSuccessful() {
+						logger.Info("Waiting for netserver to come up")
+					}
+					return res.WasSuccessful()
+				}, "netserver did not come up in time", &helpers.TimeoutConfig{Timeout: 20 * time.Second})
+				Expect(err).Should(BeNil())
```

Comment:
I would add here an assert message, will provide context in the Junit output.

Reply:
Done. It was already pretty good to debug when I tested it:
```
RuntimeConnectivityInVethModeTest
/Users/tgraf/dev/go/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:378
  Basic Connectivity test
  /Users/tgraf/dev/go/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:378
    Test NAT46 connectivity between containers [BeforeEach]
    /Users/tgraf/dev/go/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:410

    Expected
        <*errors.errorString | 0xc00033abe0>: {
            s: "Timeout reached: netserver did not come up in time",
        }
    to be nil

    /Users/tgraf/dev/go/src/github.com/cilium/cilium/test/runtime/connectivity.go:104
```